docs/guide/nameservice/04-keeper.md

```diff
@@ -1,13 +1,24 @@
 ---
 order: 4
+description: Define keepers for the nameservice module. 
 ---
 
 # Keeper
 
-The main core of a Cosmos SDK module is a piece called the `Keeper`. It handles interaction with the store, has references to other keepers for cross-module interactions, and contains most of the core functionality of a module.
+> The main core of a Cosmos SDK module is a piece called the keeper. The keeper handles interactions with the store, has references to other [keepers](https://docs.cosmos.network/master/building-modules/keeper.html) for cross-module interactions, and contains most of the core functionality of a module.
+
+## Define Keepers for the Nameservice Module 
+
```

Comment:
we can benefit from a bit more context about keepers

Reply:
Keeper is part of the SDK responsible for writing data to the store. Each module uses "its own keeper".